docs/discord/discord-readme.md

```diff
@@ -58,14 +58,10 @@ TEXT CHANNELS: These consist of the server's main text channels, that are for di
 
 - **#ideas**: got an idea / suggestion for the community? We love hearing new ideas :nerd:
 
-- **#good-first-issues**: get started with open source and get your :green_square: - opportunities are provided for new contributors - discussions not to happen here as this is just for people to share links, please discuss in #first-timers
-
-- **#first-timers**: discussion about getting :green_square: and also the issues that are mentioned in the #good-first-issues
+- **#first-timers**: discussion about getting :green_square:
 
 - **#hackathons**: the prime area to get notified about upcoming hackathons, and discussing about them
 
-- **#opensource**: discussing interesting open-source news and information, that may or may not be related to EddieHubCommunity
-
 - **#public-speaking**: improve your soft skills, public speaking, and build communities
 
 TECH: Channels regarding specific areas
```

Comment:
At line 79 in Alerts there is a channel `#bot-log`, I don't see that in discord. Should that remain or removed?